src/main/java/org/five/sonarqubot/events/MessageServiceImpl.java

```diff
@@ -10,6 +10,11 @@ public class MessageServiceImpl implements MessageService {
     public Mono<Message> onlyCodeMessages(Message eventMessage) {
         return Mono.just(eventMessage)
                 .filter(message -> message.getAuthor().map(user -> !user.isBot()).orElse(false))
-                .filter(message -> message.getContent().equalsIgnoreCase("alex"));
+                .filter(message -> message.getContent().startsWith("```") && message.getContent().endsWith("```"));
     }
-}
+
+    @Override
+    public Mono<String> getCode(Message message) {
+        return Mono.just(message.getContent().split("```")[1].trim());

```

Comment:
This code assumes there is only one codeblock in the message.
If a user sends badly formatted messages that for example contains:

\`\`\`
hejhejhej
\`\`\`
/* valid code */
\`\`\`

...then our solution would analyze "hejhejhej".

Reply:
You decide if this should be fixed right now.
We could simply make a Trello task to fix it later.